xml/Microsoft.CLRAdmin/IDataObject.xml

```diff
@@ -178,11 +214,24 @@
         <Parameter Name="b" Type="Microsoft.CLRAdmin.STGMEDIUM" RefType="ref" />
       </Parameters>
       <Docs>
-        <param name="pFormatEtc">To be added.</param>
-        <param name="b">To be added.</param>
-        <summary>This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.</summary>
-        <returns>To be added.</returns>
+        <param name="pFormatEtc">A pointer to the <see cref="T:Microsoft.CLRAdmin.FORMATETC" /> defining the format, medium, and target device for calls to retrieve data.</param>
+        <param name="b">A pointer to the <see cref="T:Microsoft.CLRAdmin.STGMEDIUM" /> indicating the storage medium containing the returned data.</param>
+        <summary>Renders the data that is described in the specified pFormatEtc structure and transfers it to the <paramref name="b" /> parameter.</summary>
```

Comment:
can you use the description from https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.comtypes.idataobject for this and the GetDataHere summaries (adapting the API paths)? Otherwise, they look exactly the same.

![image](https://user-images.githubusercontent.com/12971179/70506195-939fb280-1ade-11ea-8b63-2b7a0b25589e.png)